getting-started/templates/systemlink-values.yaml

```diff
@@ -795,35 +795,6 @@ fileingestion:
       nginx.ingress.kubernetes.io/proxy-request-buffering: "off"
       nginx.ingress.kubernetes.io/proxy-buffering: "off"
 
-  storage:
```

Comment:
This section was previously deleted and moved to cluster-specific configuration files for AWS and on-prem. A subsequent merge incorrectly restored it. I have removed it once again.